src/wp-admin/contribute.php

```diff
@@ -67,10 +67,10 @@
 	<div class="about__section has-2-columns">
 		<div class="column is-vertically-aligned-center">
 			<ul>
-				<li><?php _e( 'Pick a team' ); ?></li>
+				<li><?php _e( 'Choose a team' ); ?></li>
 				<li><?php _e( 'Follow its news to see how you can participate' ); ?></li>
-				<li><?php _e( "Attend team's meetings" ); ?></li>
-				<li><?php _e( 'Ask for Good First issue to solve' ); ?></li>
+				<li><?php _e( "Attend team&#8217;s meetings" ); ?></li>
```

Comment:
```suggestion
				<li><?php _e( 'Attend team&#8217;s meetings' ); ?></li>
```
Fix the PHPCS notice

Reply:
🤷🏻‍♂️